comfy_cli/hardware.py

```
@@ -294,6 +292,10 @@ def _detect_gpu(system: str, machine: str, cpu: str | None) -> dict | None:
 
 
 def _detect_ram_bytes() -> int | None:
+    # Lazy: psutil costs more to import than the rest of this module put
+    # together, and only RAM detection needs it.
+    import psutil
```

Pylint (4.0.7)

[convention] Import outside toplevel (psutil)

(C0415)